### DISCUSSION TITLE
Concatenation of multiple trees

### DISCUSSION THREAD

**Discussion by ramnageena11** [iqtree/iqtree2#315] [Q&A]
Hi,
I am doing a Pangenome analysis (Bacteria), where 326 different gene families need to be included for phylogenetic analysis.
I have 326 (aminoacid) alignment files and want to concatenate to make a single composite phylogenetic tree (using substitution model for evolutionary rate analysis).
Pls, suggest how to use these files as input and what options shall I choose? 
Thanks
Ram

**Comment by roblanf** [comment]
Hi @ramnageena11, sorry for the slow response - we've been flat out teaching and grant writing here.

The options really depend a lot on details of your data that only you know. I hesitate to offer analysis options, because (particularly with a concatenated alignment of hundreds of genes from organisms that love to share DNA...) they can really affect the outcome.

Probably the best things are to start by working through the tutorials:

http://www.iqtree.org/doc/Quickstart
http://www.iqtree.org/doc/Tutorial
http://www.iqtree.org/doc/Advanced-Tutorial

In terms of useful advice, I would suggest:

1. Ensure all names are consistent across alignments
2. Place all alignments into a single folder
3. Use the `-p alignment_folder` command (you can see this in various places in the docs) and IQ-TREE will automatically concatenate them for you and do a partitioned analysis
4. Look at `-m MFP` options, and `-m MFP+MERGE`, and read up on the various models you can apply to partitioned datasets (particularly the way that branch lengths are accounted for). There are good papers on this which should help. 
5. Consider estimating a model for your data with QMaker (http://www.iqtree.org/doc/Estimating-amino-acid-substitution-models), or even better GTRPmix (https://www.biorxiv.org/content/10.1101/2024.03.29.587376v1). The latter allows you to use profile mixture models, which tend to give more accurate results for bacterial analyses especially (but also a lot of other things!). 
6. Look into estimating gene and site concordance factors to get a feeling for the extent to which a single tree can actually represent your data - particularly important for bacteria... (http://www.iqtree.org/doc/Concordance-Factor)

Hope some of that helps. There's potentially a lot to learn here.

Rob

**Reply by ramnageena11** [reply]
Hi Rob,
Thanks for the suggestions and explanations. I agree this is a very vast topic to learn.

I have used following:
for model selection:
---iqtree -s /"input all .aln" files -m MF -nt AUTO
## it results in best-fit model according to BIC as: LG+F+R7

For tree:
---iqtree -s /"input all .aln" files -m LG+F+R7 -bb 1000 -nt AUTO

## generated results
- .split.nex
- .contree
- .iqtree
- .treefile
- .mldist
Pls suggest if these are okay or I need to perform this again (as you suggested).

Thanks
rgds
Ram

**Comment by roblanf** [comment]
@ramnageena11 I think this is a reasonable start. My suggestions above could help you go further, would likely change the tree you will get, and your interpretation of whether a tree is a good summary of your data.

**Reply by ramnageena11** [reply]
Thanks Rob i will follow your suggestions
Meanwhile, I have another question to ask. 
The tree I got from my approach does not have "bootstrap values" (even -bb 1000 was selected while running). When I opened it in figtree, it showed "node ages" and "branch times". How to correlate it with bootstrap values?

Thanks
rgds
Ram

**Reply by roblanf** [reply]
@ramnageena11 please stick to one question per thread. This keeps the Q&A as a useful resource for others.
